README.md

```diff
@@ -136,8 +136,34 @@ kuzzle
 
 You can add your own template, just respect the naming rule : `template_name.tpl.ext`
 
-## Testing code-example
+## Testing the code-examples locally
 
-Every time a pull request is made, a build is launch with Travis and a comment is added to the PR with the URLs of the tests reports (by language).
+It's possible to play test locally by running at the root of the project
 
-It's possible to play test locally by running at the root of the project `$ sh run_test.sh -l the_language_you_want` (js, go, ...). this will launch a kuzzle stack, and play all the tests for the language specified in an appropriate container and generate a report served locally to http://localhost:3001/reports .
+```bash
+   sh run_test.sh -l <language>
+```
+
+Where `<language>` specifies the language to test (currently available languages are `js` and `go`). This will launch a Kuzzle stack, and play all the tests for the language specified in an appropriate container and generate a report served locally to http://localhost:3001/reports .
+
+There are more available options. Using `-n` will prevent the script to launch the Kuzzle stack:
+
+```bash
+   sh run_test.sh -l <language> -n
+```
+
+This is handy if you launch many times the tests and keep the stack running on the background.
+
+You can also specify a single test to be run using the `-o` option:
```

Comment:
Nitpicking: I am not very comfortable with `-o` (it sounds like output to me), why not using `-f` (file) ?

Reply:
ok, sounds better!